.travis.yml

```diff
@@ -41,7 +41,7 @@ script:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then ~/shfmt -l -w -i 2 . ; fi
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then git diff --exit-code && echo "shfmt OK" ; fi
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then shellcheck -V ; fi
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then shellcheck -e SC2034 **/*.sh && echo "shellcheck OK" ; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then shellcheck --shell=dash **/*.sh && echo "shellcheck OK" ; fi
```

Comment:
this ensures, that we test a *POSIX* shell (so the same like .sh) but with the addition, that the keyword 'local' for e.g. 'local var=value' is allowed. This will make it possible in later commits to actually *use* that keyword without shellcheck-erros if we want it (at least I want it). fun fact: there seems to be no shell in the wild which aims to be POSIX but can not work with 'local'.